src/cmd/kubectl.rs

```diff
@@ -728,17 +728,27 @@ where
     )
 }
 
-pub fn kubectl_exec_get_event<P>(
+pub fn kubectl_exec_get_event_for_pod<P>(
```

Comment:
what about other services you don't have the pod name? Example: statefulset running MySQL/Postgres...